metadata.rb

```diff
@@ -4,7 +4,7 @@
 license           'Apache 2.0'
 description       'Installs and configures haproxy'
 long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version           '4.0.2'
+version           '4.0.3'
```

Comment:
verified w/ @rshade that you can leave the bump here for this PR since he will release once this is merged, but not to bump the version in the future